addons/udes_stock_refactoring/tests/test_splitting.py

```diff
@@ -0,0 +1,649 @@
+# -*- coding: utf-8 -*-
+
+from addons.udes_stock.tests import common
+
+
+class TestSplittingBase(common.BaseUDES):
```

Comment:
We could probably rename this to Refactoring instead of splitting. Same for the file name.

Reply:
Will leave this till the very end incase further changes are needed to the tests.